src/main/steps/you-can-use-english-welsh-courts/content.ts

```diff
@@ -92,16 +92,14 @@ const en = (
     clarification: '',
   };
 
-  const enConnections: Record<JurisdictionConnections, typeof enPetRespResident | undefined> = {
+  const enConnections: Partial<Record<JurisdictionConnections, typeof enPetRespResident | undefined>> = {
     [JurisdictionConnections.PET_RESP_RESIDENT]: enPetRespResident,
     [JurisdictionConnections.PET_RESP_LAST_RESIDENT]: enPetRespLastResident,
     [JurisdictionConnections.RESP_RESIDENT]: enRespResident,
     [JurisdictionConnections.PET_RESIDENT_TWELVE_MONTHS]: enPetResidentTwelveMonths,
     [JurisdictionConnections.PET_RESIDENT_SIX_MONTHS]: enPetResidentSixMonths,
     [JurisdictionConnections.PET_RESP_DOMICILED]: enPetRespDomiciled,
     [JurisdictionConnections.RESIDUAL_JURISDICTION]: enResidualJurisdiction,
-    [JurisdictionConnections.PET_DOMICILED]: undefined,
```

Comment:
based on existing service, connection h and i arent shown on the interstitial page. first element is only shown.